packs/elastic/pack.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name : elastic
```

Comment:
And other comments which haven't been addressed yet from the old PR (https://github.com/StackStorm/st2contrib/pull/198) still apply :)